airbyte-integrations/connectors/destination-s3/src/test/java/io/airbyte/integrations/destination/s3/parquet/ParquetSerializedBufferTest.java

```diff
@@ -77,6 +78,27 @@ public void testCompressedParquetWriter() throws Exception {
     runTest(195L, 215L, config, getExpectedString());
   }
 
+  private static String resolveArchitecture() {
+    return System.getProperty("os.name").replace(' ', '_') + "-" + System.getProperty("os.arch") + "-" + System.getProperty("sun.arch.data.model");
+  }
+
+  @Test
+  public void testLzoCompressedParquet() throws Exception {
+    if (resolveArchitecture().equals("Linux-amd64-64")) {
```

Comment:
I think we should add test in the case of aarch64 || arm64 architectures since we're expecting that to work based on the Dockerfile additions.

Reply:
> aarch64 || arm64 architectures

@Phlair added test for aarch64 || arm64 architectures